src/docs/sphinx/docker_info.rst

```diff
@@ -0,0 +1,34 @@
+.. ## Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+.. ## other Serac Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+===========
+Docker Info
+===========
+
+Building a new image
+--------------------
+
+The following instructions apply to the creation of a new compiler image.
+
+    1. Start by cloning down the ``serac`` repository.  
+    #. Ensure that an Axom image exists on Dockerhub for the desired compiler.
+       If no corresponding Axom compiler image exists, it should be 
+       created before proceeding.
+    #. Go to the ``scripts/docker`` directory and run ``build_new_image.sh``, passing the compiler
+       name and version, e.g. for Clang 10, run ``./build_new_image.sh clang 10``.  Minor versions can also be specified.
```

Comment:
It's very common to have a minor version, can you provide an example here for that.